### PR TITLE
Submit two website test cases: "InstallPackageToWebSiteFromUI" & "UpdateWebSitePackageFromUI"

### PR DESCRIPTION
**Pull request by v-luzh**
## Description

Add two test cases 
1.InstallPackageToWebSiteFromUI: Install particular package to C# ASP Web Site Project from PM UI.
2.UpdateWebSitePackageFromUI: Update package for C# ASP Web Site Project in PM UI.

- Updated the "...\build\tests.dev17.vsconfig" to install Web Site Projects for testing.
- We created a bulk of C# ASP Web Site Projects with Theory.
- The test cases were added into the end of the ...\NuGet.Tests.Apex\NuGetEndToEndTests\NuGetUITestCase.cs file.

## PR Checklist

- [ ] PR has a meaningful title
- [ ] PR has a linked issue.
- [ ] Described changes

- **Tests**
  - [ ] Automated tests added

### CODE REVIEW

**Review by nkolev92 — APPROVED** [review]

File: test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetEndToEndTests/NuGetUITestCase.cs [hunk 1]
```diff
@@ -509,5 +512,75 @@ private static void WaitForFileNotExists(FileInfo file)
...
+            CommonUtility.OpenNuGetPackageManagerWithDte(VisualStudio, XunitLogger);
+            var nugetTestService = GetNuGetTestService();
+            var uiwindow = nugetTestService.GetUIWindowfromProject(project);
+            uiwindow.InstallPackageFromUI(packageName, packageVersion);
+
+            // Assert
+            CommonUtility.AssertPackageInPackagesConfig(VisualStudio, project, packageName, packageVersion, XunitLogger);
+        }
+
+        public static IEnumerable<object[]> GetWebSiteTemplates()
```
Comment (aortiz-msft) [inline test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetEndToEndTests/NuGetUITestCase.cs:540]: >public [](http://example.com/codeflow?start=8&length=6)

Does this need to be public?

File: test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetEndToEndTests/NuGetUITestCase.cs [hunk 2]
```diff
@@ -509,5 +512,75 @@ private static void WaitForFileNotExists(FileInfo file)
...
+
+            // Act
+            VisualStudio.ClearWindows();
+            uiwindow.UpdatePackageFromUI(packageName, updateVersion);
+
+            // Assert
+            CommonUtility.AssertPackageInPackagesConfig(VisualStudio, project, packageName, updateVersion, XunitLogger);
+        }
+
+        public static IEnumerable<object[]> UpdateWebSiteTemplates()
```
Comment (aortiz-msft) [inline test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetEndToEndTests/NuGetUITestCase.cs:577]: >public [](http://example.com/codeflow?start=8&length=6)

Same.

File: test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetEndToEndTests/NuGetUITestCase.cs (hunk 1 above)
Comment (v-luzh) [reply test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetEndToEndTests/NuGetUITestCase.cs:540]: OK, we will close this one and recreate a new draft PR to public dev.

**Comment by nkolev92** [comment]
Hey @v-luzh 

You can push changes to branches, retargeting etc, you don't have to close the PRs. 
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-base-branch-of-a-pull-request

**Comment by v-luzh** [comment]
Hi @nkolev92, thanks for your instructions and we will retarget the branch instead of closing the PR next time. But we just saw your comment after closing this PR. Sorry about it.

And we submitted another PR [4837](https://github.com/NuGet/NuGet.Client/pull/4837) before the holiday. Cloud you please help to confirm the new one? Thanks a lot!

**Pull request closed without merging**